### PR TITLE
Add applied control duplication

### PR DESCRIPTION
**Pull request by monsieurswag**

### CODE REVIEW

**Review by Mohamed-Hacene — CHANGES_REQUESTED** [review]
- [ ] Use a slot for duplication button inside the DetailView
- [ ] Add a checkbox to bring the evidence if needed

**Comment by monsieurswag** [comment]
We don't need a to use a slot now, the code can be easily adapted to add duplication for other objects.
The checkbox to bring the evidences has been added, check its behavior in the backend (some evidences are just linked while other are duplicated depending on their folder)

File: backend/core/views.py [hunk 1]
```diff
@@ -1039,6 +1043,47 @@ def get_timeline_info(self, request):
...
+            effort=applied_control.effort,
+            cost=applied_control.cost,
+        )
+        if data["duplicate_evidences"]:
+            duplicate_related_objects(
+                applied_control, duplicate_applied_control, new_folder, "evidences"
+            )
+            duplicate_applied_control.save()
+
+        return Response({"results": "applied control duplicated"})
```
Comment (nas-tabchiche) [inline backend/core/views.py]: Let's return the actual controls as JSON

**Review by Mohamed-Hacene — DISMISSED** [review]
- Field `has_evidence` (and `eta_missed` by the way) should be excluded. We cannot add evidence from the applied control anymore.
image
- The checkbox has unrelated text with this PR
image

**Comment by ab-smith** [comment]
Pensez au squash au moment du merge

**Review by eric-intuitem — APPROVED** [review]
LGTM

**Review by eric-intuitem — APPROVED** [review]
LGTM

**Pull request merged**